beginner_source/blitz/tensor_tutorial.py

```diff
@@ -107,21 +153,29 @@
 # .. note::
 #     Any operation that mutates a tensor in-place is post-fixed with an ``_``.
 #     For example: ``x.copy_(y)``, ``x.t_()``, will change ``x``.
-#
-# You can use standard NumPy-like indexing with all bells and whistles!
+
+###############################################################
+# Similar to NumPy, tensors can be index using the standard
+# Python ``x[i]`` syntax, where ``x`` is the array and ``i`` is the selection.
+# 
+# That said, you can NumPy-like indexing with all its bells and whistles!
 
 print(x[:, 1])
 
 ###############################################################
-# Resizing: If you want to resize/reshape tensor, you can use ``torch.view``:
+# Resizing your tensors might be necessary for your data.
+# If you want to resize or reshape tensor, you can use ``torch.view``:
+
 x = torch.randn(4, 4)
 y = x.view(16)
 z = x.view(-1, 8)  # the size -1 is inferred from other dimensions
 print(x.size(), y.size(), z.size())
 
 ###############################################################
-# If you have a one element tensor, use ``.item()`` to get the value as a
-# Python number
+# You can access the Python number-value of a one-element tensor using ``.item()``.
+# If you have a multidimensional tensor, see the 
+# `tolist() https://pytorch.org/docs/stable/tensors.html#torch.Tensor.tolist` method.
```

Comment:
I think you might need to add brackets and an underscore at the end for the hyperlink to work:
`tolist() <https://pytorch.org/docs/stable/tensors.html#torch.Tensor.tolist>`_

https://devguide.python.org/documenting/#hyperlinks
https://github.com/pytorch/tutorials/blob/master/advanced_source/dynamic_quantization_tutorial.py#L37